### ISSUE TITLE
Does the lib support Pythton 2?

### ISSUE THREAD

**Issue by dcaragao** [django-ses/django-ses#141]
Hello,

I am currently trying to confirm a SNS subscription, but I always get:

```

TypeError
unicode argument expected, got 'str'
django_ses/utils.py in _get_bytes_to_sign at line 175
        outbytes = StringIO()
        for field_name in fields_to_sign:
            field_value = smart_str(self._data.get(field_name, ''),
                                    errors="replace")
            if field_value:
                outbytes.write(field_name)
                outbytes.write("\n")
                outbytes.write(field_value)
                outbytes.write("\n")
        return outbytes.getvalue()
```
I believe it happens because in Python 2.x strings are 'str' by default (what makes it necessary to refer to them as `u'somestring'`), while in Python 3.x strings are 'unicode' by default. Could this be considered an issue or the django-ses **intentionally** does not support Python 2?

**Comment by pcraciunoiu** [comment]
It should work with 2. It might be a recent fix that broke this--did you try the previous version?

Also, if you want to write a test for 2 and one for 3, that would be great! Even better if you want to submit a fix.

**Comment by pcraciunoiu** [comment]
Closed in #143 and follow up in #167.
